### PR TITLE
MAT-6746 pulled updated models(remove cms id at measure level) & test case fix

### PR DESCRIPTION
**Pull request by adongare**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-6746](https://jira.cms.gov/browse/MAT-6746)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/test/java/gov/cms/mat/cql_elm_translation/config/logging/RequestResponseLoggingInterceptorTest.java [hunk 1]
```diff
@@ -26,26 +26,21 @@ class RequestResponseLoggingInterceptorTest {
   @Mock private ClientHttpRequestExecution mockClientHttpRequestExecution;
 
   @Test
-  void test() {
+  void testHttpRequestIntercept() throws IOException {
```
Comment (adongare) [inline src/test/java/gov/cms/mat/cql_elm_translation/config/logging/RequestResponseLoggingInterceptorTest.java:29]: to fix the Jenkins build that failed previously. https://semanticbits.slack.com/archives/CNU5AEYMV/p1712266682960979
I'm not sure how useful is this interceptor because it only works if the log level is DEBUG .

**Review by gregory-akins — APPROVED** [review]
